install_dep_packages_acc.sh

```diff
@@ -1,17 +1,11 @@
-# Copyright (c) 2021 Intel Corporation.
+#!/bin/bash
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at:
+# Copyright 2023 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Any reason for moving to SPDX-License from  Apache License ?

Reply:
ipsec-recipe is not licensed under Apache as per this file: https://github.com/5abeel/ipsec-recipe/blob/main/LICENSE